numpy/core/tests/test_numerictypes.py

```diff
@@ -102,99 +102,99 @@ def normalize_descr(descr):
 #    Creation tests
 ############################################################
 
-class create_zeros(object):
+class CreateZeros(object):
     """Check the creation of heterogeneous arrays zero-valued"""
 
     def test_zeros0D(self):
         """Check creation of 0-dimensional objects"""
         h = np.zeros((), dtype=self._descr)
-        self.assertTrue(normalize_descr(self._descr) == h.dtype.descr)
-        self.assertTrue(h.dtype.fields['x'][0].name[:4] == 'void')
-        self.assertTrue(h.dtype.fields['x'][0].char == 'V')
-        self.assertTrue(h.dtype.fields['x'][0].type == np.void)
+        assert_(normalize_descr(self._descr) == h.dtype.descr)
+        assert_(h.dtype.fields['x'][0].name[:4] == 'void')
+        assert_(h.dtype.fields['x'][0].char == 'V')
+        assert_(h.dtype.fields['x'][0].type == np.void)
         # A small check that data is ok
         assert_equal(h['z'], np.zeros((), dtype='u1'))
 
     def test_zerosSD(self):
         """Check creation of single-dimensional objects"""
         h = np.zeros((2,), dtype=self._descr)
-        self.assertTrue(normalize_descr(self._descr) == h.dtype.descr)
-        self.assertTrue(h.dtype['y'].name[:4] == 'void')
-        self.assertTrue(h.dtype['y'].char == 'V')
-        self.assertTrue(h.dtype['y'].type == np.void)
+        assert_(normalize_descr(self._descr) == h.dtype.descr)
+        assert_(h.dtype['y'].name[:4] == 'void')
+        assert_(h.dtype['y'].char == 'V')
+        assert_(h.dtype['y'].type == np.void)
         # A small check that data is ok
         assert_equal(h['z'], np.zeros((2,), dtype='u1'))
 
     def test_zerosMD(self):
         """Check creation of multi-dimensional objects"""
         h = np.zeros((2, 3), dtype=self._descr)
-        self.assertTrue(normalize_descr(self._descr) == h.dtype.descr)
-        self.assertTrue(h.dtype['z'].name == 'uint8')
-        self.assertTrue(h.dtype['z'].char == 'B')
-        self.assertTrue(h.dtype['z'].type == np.uint8)
+        assert_(normalize_descr(self._descr) == h.dtype.descr)
+        assert_(h.dtype['z'].name == 'uint8')
+        assert_(h.dtype['z'].char == 'B')
+        assert_(h.dtype['z'].type == np.uint8)
         # A small check that data is ok
         assert_equal(h['z'], np.zeros((2, 3), dtype='u1'))
 
```

Comment:
Above here, I did not check that the classes are named `Test...` not sure it is important.